pkg/test/requester/node_selection_test.go

```diff
@@ -63,13 +63,10 @@ func (s *NodeSelectionSuite) SetupSuite() {
 			},
 		},
 	}
-	requesterConfig, err := node.NewRequesterConfigWith(
-		node.RequesterConfigParams{
-			NodeRankRandomnessRange: 0,
-			OverAskForBidsFactor:    1,
-		},
-	)
```

Comment:
No ides how this ever worked before `NodeRankRandomnessRange` cannot be `0` or the code that uses it panics, somehow it never panicked, meaning the setting of these values were likely ineffectual